docs/source/model_definition.md

```diff
@@ -67,16 +117,25 @@ model_body = SequentialBlock(
 )
 ```
 
+## Prediction Head (Output Block)
+
+Following the input and transformer blocks, the model outputs its predictions.
+Transformers4Rec supports the following prediction heads, which can have multiple losses and can be combined for multi-task learning and multiple metrics:
+
+- **Next Item Prediction**: Predicts next items for a given sequence of interactions.
+During training, the prediction can be the next item or randomly selected items depending on the masking scheme.
+For inference, the intended purpose is to always predict the next interacted item.
+Cross-entropy and pairwise losses are supported.
 
-## Prediction head (Output Block)
-Following the input and transformer blocks the model outputs its predictions.  The library supports the following prediction heads which can have multiple losses and can be combined for multi-task learning and multiple metrics.
+- **Classification**: Predicts a categorical feature using the whole sequence.
```

Comment:
In T4Rec, we are only supporting the `Binary Classification` case. So perhaps, we can be more specific in the description? 
```suggestion
- **Binary Classification**: Predicts a binary feature using the whole sequence.
```